### DISCUSSION TITLE
Documentación para los Tests

### DISCUSSION THREAD

**Discussion by gs-Manuel** [Arquisoft/faq#69] [General]
¿Es necesario añadir una sección para los tests en la documentación arc42? 
En caso de que sí, que deberíamos indicar en esta sección?

**Comment by labra** [comment]
En las trasparencias de la sesión de prácticas de la semana pasada os indicamos que podíais añadir un capítulo a la documentación de arc42 con una sección sobre el informe de pruebas. 

En esa sección podéis incluir información sobre las pruebas que hayáis realizado: pruebas unitarias, pruebas de aceptación, pruebas de carga, otras pruebas, así como los resultados de las mismas y las posibles conclusiones o mejoras realizadas en base a dichos resultados.
